src/wasm.rs

```diff
@@ -1336,9 +1336,13 @@ impl WasmModule {
 
     fn process_table_element_section(&mut self, p: &mut Parser) -> ProcessState {
         match p.read() {
-            &ParserState::BeginActiveElementSectionEntry(table_id) => {
-                ProcessState::TableElementEntry { table_id }
-            }
+            &ParserState::BeginElementSectionEntry { table, ty: _ } => match table {
+                ElemSectionEntryTable::Active(table_id) => {
+                    ProcessState::TableElementEntry { table_id }
+                }
+                ElemSectionEntryTable::Passive => ProcessState::TableElementSection,
+                ElemSectionEntryTable::Declared => ProcessState::TableElementSection,
```

Comment:
What does this mean?

Reply:
Since I'm not 100% sure, I've replaced these arms with unimplemented panics. I'll circle back once we're current.